### PR TITLE
Added subclasses of intrafusal muscle

### LINKED ISSUE #848: Add spindle component 'nuclear chain fiber' and 'nuclear bag fiber'
Terms for subclasses of beta motor neurons. 
See #845

### PR DESCRIPTION
**Pull request by shawntanzk**
fixes #848

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27043,6 +27047,38 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
...
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023003 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023003 "nuclear chain fiber")
+SubClassOf(obo:CL_4023003 obo:CL_0008047)
+
+# Class: obo:CL_4023004 (nuclear bag fiber)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") Annotation(oboInOwl:hasDbXref "PMID:139469") obo:IAO_0000115 obo:CL_4023004 "A type of intrafusal muscle fiber that lies in the center of a muscle spindle. Nuclei are clustered centrally and give the equatorial region a swollen appearance. They are associated with associated with dynamic gamma motor neurons, and the stretching of the equatorial region of the nuclear bag fibers results in an increase in the firing rate of type Ia sensory fibers.")
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:27059]: Will write a ticket to add sensory fibers

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -27043,6 +27047,38 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
 SubClassOf(obo:CL_3000004 obo:CL_0000101)
 SubClassOf(obo:CL_3000004 obo:CL_0000111)
 
+# Class: obo:CL_4023003 (nuclear chain fiber)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") Annotation(oboInOwl:hasDbXref "PMID:139469") obo:IAO_0000115 obo:CL_4023003 "A type of intrafusal muscle fiber that have nuclei arranged in a linear row, and the equatorial region is not obviously expanded. are responsible for the detection of changes in muscle length. They are innervated by static gamma motor neurons and are principally associated with type II sensory fiber")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023003 <http://orcid.org/0000-0001-7258-9596>)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27053]: Suggested edits to text def:

A type of intrafusal muscle fiber that has nuclei arranged in a linear row.  Unlike nuclear bag fibers, the equatorial region of these fibers (in the centre of the spindle) is not expanded. These fibers are responsible for the detection of changes in muscle length. They are innervated by static gamma motor neurons and are principally associated with type II sensory fibers.

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -27043,6 +27047,38 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
...
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023005 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023005 "bag1 fiber")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023005 "nuclear bag1 intrafusal fiber")
+AnnotationAssertion(rdfs:label obo:CL_4023005 "dynamic nuclear bag fiber")
+SubClassOf(obo:CL_4023005 obo:CL_4023004)
+
+# Class: obo:CL_4023006 (static nuclear bag fiber)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") obo:IAO_0000115 obo:CL_4023006 "They are sensitive to only changes in muscle length but not the rate of that change.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023006 <http://orcid.org/0000-0001-7258-9596>)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27076]: A nuclear bag fiber that is sensitive only changes in muscle length but not the rate of that change.

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -27043,6 +27047,38 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") Annotation(oboInOwl:hasDbXref "PMID:139469") obo:IAO_0000115 obo:CL_4023004 "A type of intrafusal muscle fiber that lies in the center of a muscle spindle. Nuclei are clustered centrally and give the equatorial region a swollen appearance. They are associated with associated with dynamic gamma motor neurons, and the stretching of the equatorial region of the nuclear bag fibers results in an increase in the firing rate of type Ia sensory fibers.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023004 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023004 "nuclear bag fiber")
+SubClassOf(obo:CL_4023004 obo:CL_0008047)
+
+# Class: obo:CL_4023005 (dynamic nuclear bag fiber)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") obo:IAO_0000115 obo:CL_4023005 "They are sensitive mainly to the rate of change in muscle length.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023005 <http://orcid.org/0000-0001-7258-9596>)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27067]: A nuclear bag fiber that is sensitive mainly to the rate of change in muscle length.

**Review by dosumis — CHANGES_REQUESTED** [review]
Looks good.  Have requested some changes in def text.  We should look over guidance for writing text defs.

**Comment by paolaroncaglia** [comment]
Hi @dosumis and @shawntanzk (pleasure to meet you Shawn!),

> Looks good. Have requested some changes in def text. We should look over guidance for writing text defs.

If it's helpful, Chris Mungall wrote a blog post on this some time ago, https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/

Cheers, Paola.

**Review by dosumis — APPROVED** [review]

**Pull request merged**